libraries/AP_Notify/ToneAlarm.cpp

```diff
@@ -226,18 +226,38 @@ void AP_ToneAlarm::update()
         return;
     }
 
-    if (AP_Notify::events.compass_cal_saved) {
+    if (AP_Notify::events.compass_cal_saved ||
+        AP_Notify::events.temp_cal_saved) {
         play_tone(AP_NOTIFY_TONE_QUIET_READY_OR_FINISHED);
         return;
     }
 
-    if (AP_Notify::events.compass_cal_failed) {
+    if (AP_Notify::events.compass_cal_failed ||
+        AP_Notify::events.temp_cal_failed) {
         play_tone(AP_NOTIFY_TONE_QUIET_NEG_FEEDBACK);
         return;
     }
 
-    // don't play other tones if compass cal is running
-    if (AP_Notify::flags.compass_cal_running) {
+    if (AP_Notify::events.initiated_temp_cal) {
+        play_tone(AP_NOTIFY_TONE_QUIET_NEU_FEEDBACK);
```

Comment:
```suggestion
        play_tone(AP_NOTIFY_TONE_QUIET_NEG_FEEDBACK);
```

Reply:
why? I made it match the pattern for compass